app/models/review.rb

```diff
@@ -0,0 +1,32 @@
+# == Schema Information
+#
+# Table name: reviews
+#
+#  id            :bigint           not null, primary key
+#  text          :text
+#  reviewer_name :string(40)
+#  rating        :integer
+#  restaurant_id :integer
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#
+
+class Review < ApplicationRecord
+  # Associations
+  belongs_to :restaurant, foreign_key: :restaurant_id
+
+  # Scopes
+  scope :sorted_by_created_at, lambda { order(created_at: :desc) }
+
+  # Validations
+  validates :text,
+            presence: true
+  validates :reviewer_name,
+            presence: true,
+            length: {maximum: 40}
```

Comment:
minor styling and consistency:
we tend to add spaces on both sides of the ruby hashes
{ maximum: 40 }

once installing rubocop you will get alerts about these things.